src/clib/lib/include/ert/enkf/field.hpp

```diff
@@ -17,7 +17,6 @@
 void field_scale(field_type *field, double scale_factor);
 extern "C" double field_iget_double(const field_type *, int);
 extern "C" double field_ijk_get_double(const field_type *field, int, int, int);
-float field_iget_float(const field_type *, int);
```

Comment:
I thought that field was only float and not double? :shrug:

Reply:
Fields contain `float`s but the rest of ERT likes to cast it to `double`, so we only keep the other version of `iget`.